boot/seal.go

```diff
@@ -102,6 +102,8 @@ type sealKeyToModeenvFlags struct {
 	SeedDir string
 	// Unlocker is used unlock the snapd state for long operations
 	StateUnlocker Unlocker
+	// UseTokens disables key file to save key datas and instead uses tokens of key slots
+	UseTokens bool
```

Comment:
```suggestion
	// UseTokens indicates that the key should be saved to the token of a
	// corresponding key slot
	UseTokens bool
```

Reply:
"UseTokens indicates that key data should be saved to the tokens of key slots. If not, they will be saved to key files."?